sky/server/constants.py

```diff
@@ -7,7 +7,7 @@
 # API server version, whenever there is a change in API server that requires a
 # restart of the local API server or error out when the client does not match
 # the server version.
-API_VERSION = '4'
+API_VERSION = '5'
```

Comment:
Technically, there is no backwards-incompatible change here. However, since the compatibility check relies on server-side changes, a stale API server will silently fail/hit #5311 unless we force an error by bumping the version.